rebasehelper/cli.py

```diff
@@ -240,6 +248,17 @@ def __getattr__(self, name):
 class CliHelper(object):
 
     @staticmethod
+    def set_colors(app):
+        """Set app.conf.colors variable"""
+        if not os.isatty(1) or os.environ.get('TERM') == 'dumb'\
```

Comment:
How about using `sys.stdout.fileno()` instead of `1`? It doesn't always need to be like that.